krm/filter.go

```diff
@@ -40,6 +44,9 @@ func DefaultNodeHandler(_, currentRef, nextRef string, opts Options) (string, er
 		return currentRef, err
 	}
 
+	if opts.Part == "tag" {
```

Comment:
I am not sure how to handle the tag in this situation. 

One way could be to have the digest as a seperate part to select from. Then we also need a way to combine parts into a single output field.

Or everything after the image is considered the tag or _identifier_ how the container package calls it.  

On top of that, the built in decoder actuall generate `<tag>@<digest>` identifiers. This is technically not correct. Registiries will ignore the tag but its nice for informational purposes.